### ISSUE TITLE
Zoom performs poorly outside of the map

### ISSUE THREAD

**Issue by rleir** [nextstrain/auspice#1116]
**Current Behavior**  
A clear and concise description of what is happening / what the bug is.  
Zoom using ctrl+scroll works nicely in the map area. Outside of that it seems broken.

**Expected behavior**  
A clear and concise description of what you expected to happen instead.  
Smooth scrolling in the tree and map but nowhere else.

**How to reproduce**  
Steps to reproduce the current behavior:  
1. I am looking at this dataset but it is the same for others: http://localhost:4000/flu/seasonal/yam/ha/3y  
2.  In the map area, ctrl+scroll using the mouse scroll wheel works nicely to zoom
3.  In the tree area, it causes zoom but with bad lag.
4. ctrl+scroll then stop. You see two-second intervals between changed zoom levels
5. the zooming seems to affect the whole screen, not just the tree.

**Possible solution**  
(optional)  
Throttling and .. ?

**Your environment: if browsing Nextstrain online**  
 - Operating system:  
 - Browser:  

**Your environment: if running Nextstrain locally**  
 - Operating system:   Ubuntu 19.10
 - Browser:    Chrome
 - Version (e.g. `auspice 2.7.0`):  master 2.14.0

**Additional context**

**Comment by jameshadfield** [comment]
The "ctrl+scroll" zoom functionality is only implemented in auspice for the map (as you scroll through the page normally, if your cursor is over the map, you'll see an overlay over the map indicating this). Using "ctrl+scroll" anywhere other than inside the map is not caught by auspice and is instead triggering your browser to increase the zoom level of the entire webpage (the exact key combo might be browser specific). To zoom into parts of the tree you can click on internal branches. We could do a better job of highlighting this functionality.

**Comment by rleir** [comment]
Hi James 
Leaflet maps commonly provide zoom without the Ctrl, just scroll. I would recommend that to make it less confusing.
Thanks -- Rick

**Comment by emmahodcroft** [comment]
Hi @rleir ,
Thanks for reaching out and sorry this was confusing! 

We used to have zoom without CTRL, however, we found this often led to an experience where users struggled to scroll down the webpage (to look at the entropy view, or use the filters) because if their mouse ended up over the map while scrolling, it switches to zooming the map rather than scrolling the page. This is understandably quite frustrating. Thus, we've implemented the zoom to work with CTRL over the map, so that one only zooms the map when this is the intended action, otherwise allowing smooth scrolling up and down the page. 

I hope this helps make our use of the CTRL+scroll on the map clear!
